packages/apps/plugins/plugin-tree/src/components/Tree/Tree.stories.tsx

```diff
@@ -0,0 +1,100 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import '@dxosTheme';
+
+import { deepSignal } from 'deepsignal/react';
+// eslint-disable-next-line
+import React, { useState } from 'react';
+
+import { PublicKey } from '@dxos/keys';
+import { DensityProvider } from '@dxos/react-ui';
+
+import { Tree } from './Tree';
+
+const Story = () => {
+  const [root] = useState(
+    deepSignal({
```

Comment:
`deepSignal` is already recursive, all of the calls to `deepSignal` inside this top one are redundant.

Reply:
IIUC, it’s an antipattern to use `deepSignal` and `useState` together, since they achieve the same goal slightly differently. Consider either instantiating a `deepSignal` and passing it as a prop to a component or using `useState` on its own.

Reply:
Ah nice catch/lesson. Thanks.